var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -18,6 +18,7 @@ class SuperluDist(CMakePackage):
 
     version('develop', branch='master')
```

Comment:
```suggestion
    version('master', branch='master')
```

Reply:
Its one of those things - if want all xsdk packages to use the same spack notation `package@develop` or not.

And some packages might user `master` branch, some might user `develop` branch
If needed - I can keep both.

```
version('develop', branch='master')
version('master', branch='master')
```

Reply:
You can keep it that way for now, but I'm never going to remember that your packages are special and I'll continue to remind you to switch everything to master 😛